README.md

```diff
@@ -131,6 +139,116 @@ To start either of the interfaces in development mode:
   - Unit-tests and UI-tests for all interfaces; Run with `npm test` in each `interfaces/*`-folder.
   - Unit-tests and integration-tests for all services; Run with `npm test` in each `services/*`-folder.
 
+## Common problems with Local Environment set-up
+If the swagger UI is not accessible after installing docker and setting up services, we can take following steps to debug:
+1. `docker ps` to list running containers and their statuses
+2. `docker logs -f <container-name>` to check their statuses
+
+- If the errors are related to not being able to access/connect to 'database' then resetting the database by using `dropSchema: true` in `ormconfig.ts`.
+
+- In order to access docker container itself, execute command `sudo docker exec -it <container-name> sh` and verify installation of npm modules.
+
+
+## Unit Tests
```

Comment:
There is a section-header "Testing" at line 136: https://github.com/global-121/121-platform/pull/1542/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R136

I think that should be 'moved down here', so we get the order:

1. getting started / setup dev-environment
2. common issues (or as a sub-section of the previous)
3. testing
4. unit-tests (or as a sub-section, might be even better)